src/main/java/com/amplitude/exception/AmplitudeInvalidAPIKeyException.java

```diff
@@ -0,0 +1,30 @@
+package com.amplitude.exception;
+
+public class AmplitudeInvalidAPIKeyException extends Exception {
```

Comment:
could we have an `AmplitudeException` then let this extends `AmplitudeException`